### ISSUE TITLE
xliffmerge creates empty json files?

### ISSUE THREAD

**Issue by matheo** [martinroob/ngx-i18nsupport#18]
Hi Martin!
Thanks a lot for this tool, and the TinyTranslator holy cow! super useful!

I wonder why after `xliffmerge` execution there remains some empty json files side by side with the localized files, like:
```
- messages.en-us.json
- messages.en-us.xlf
- messages.es.json
- messages.es.xlf
- messages.xlf
```
I will give you a hand with the TinyTranslator interface when i have spare time!
Thanks again!

**Comment by martinroob** [comment]
Hello,
the json files are created for usage with ngx-translate.
They should not be created, if you set

supportNgxTranslate: false

in the settings (otherwise, this would be a bug).
There is a wiki page about it [ngx translate usage](https://github.com/martinroob/ngx-i18nsupport/wiki/ngx-translate-usage).
But it is a feature that will die in the near future, when angulars i18n supports things like this natively.

**Comment by martinroob** [comment]
When there are no texts marked for ngx translate usage, there should be no empty json file, but just no file.
So I do not close this issue, will fix it soon.

**Comment by matheo** [comment]
Got it :)
Thanks for the great work @martinroob!

**Comment by martinroob** [comment]
closed with 0.4

**Comment by matheo** [comment]
Thanks @martinroob 

I will elaborate a feature request to update the source, and maybe change the state when the unit was changed, so changes can be tracked easily.
